src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/start-free-scan/StartFreeScanView.tsx

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image from "next/image";
+import ImageCityScape from "./images/city-scape.svg";
+import styles from "../dataBrokerProfiles.module.scss";
+import { Button } from "../../../../../../../../components/server/Button";
+import { getL10n } from "../../../../../../../../functions/server/l10n";
+import { FixView } from "../../FixView";
+import {
+  StepDeterminationData as StepDeterminationData,
+  getNextGuidedStep,
+} from "../../../../../../../../functions/server/getRelevantGuidedSteps";
+
+export type Props = {
+  data: StepDeterminationData;
+  subscriberEmails: string[];
+};
+
+export function StartFreeScanView(props: Props) {
+  const l10n = getL10n();
+
+  return (
+    <FixView
+      data={props.data}
+      subscriberEmails={props.subscriberEmails}
+      nextStepHref={getNextGuidedStep(props.data, "Scan")?.href ?? ""}
+    >
+      <div className={styles.contentWrapper}>
```

Comment:
All the content in here is the same as it was in `page.tsx`, but now wrapped in `<FixView>`, and with the data passed in as props.